modules/special-rules/pages/holo-fields.adoc

```diff
@@ -0,0 +1,23 @@
+= Holo-fields
+
+This unique technology projects a field that can hide an Eldar titan in a colourful but diffuse and baffling appearance.
+This makes it almost impossible for the enemy to pinpoint the precise location of the titan -- it can thwart even the most advanced targetting systems.
+
+However, Eldar titans are of light construction -- they are built for speed and agility, not resilience.
+
+---
+
+Where your unit has the {holo-fields} ability:
+
+* Each time the unit suffers a hit, from any source, roll a die for a {holo-fields} save:
+ ** *2+*: Successful save, ignore the hit -- it causes no damage.
+ (But place Blast markers as normal.)
+ ** *1*: Unsuccessful save, apply the hit as normal.
```

Comment:
Personally, I'm not a fan of this change. It feels very wordy without being clearer than the previous 'on a roll of X+, do Y' structure. 

Also, the 'saving roll' part of Save and Holo-fields is exactly the same ability with a different target number, so they ought to use the same structure.

Reply:
The rest of the page looks better, though!

Reply:
I'm open to other ideas, but keep in mind that in the previous iteration the term "unsaved" came out of nowhere, with no reference to 'saving'. Also you have to be careful in how you describe things like 'critical hit roll' and similar, because (a) the correct term is critical damage, and (b) it can be ambiguous as to whether you mean to roll to see if you get critical damage, or to roll to see what the critical damage result is.

So, it may be more wordy and have room for improvement, but I think that it **is** clearer for these reasons. I don't make these changes because I have some arbitrary preference that I want to impose -- I make them because I perceive a problem in the structure, logic and/or phrasing (and some other reasons too to do with best practices, consistency, etcetera).

Reply:
Anyway, let's mark this as resolved. We can talk about which structure is a better fit at another time, and not let it hold up the pull request and branch merge.

I'd like to start on new content at some point!

Reply:
Okay, let's mark resolved and move on. But so you have some idea where I'm coming from with these things:

Previously:

>* Each time the unit suffers a hit, from any source, roll a die. On a roll of 2+, the hit is ignored and causes no damage.
>* The {holo-fields} ability applies to attacks that ignore void shields and power fields.
>* Any unsaved hit on the unit is automatically a critical hit. Attacks which already automatically cause critical hits only cause one critical hit.
>
>NOTE: Saved hits still count as hits for placing Blast markers.

Yes, the text from Save would've worked to improve that first line, to be consistent and remove passive language etcetera --  **in isolation**. But it isn't in isolation.

Some of issues that the rest of the text poses:

1. 'Unsaved hit' (and then later, 'Saved hits') -- what's that? There is no prior mention of anything to do with save, saving, saved or similar. Yes, most players will know what we mean... but that's assumption, not explanation.
2. The *Any unsaved hit ..." bullet doesn't flow logically from the lead-in sentence. And even if it did, I think this one works better as a note, on the whole.
3. Given the restructure, we can eliminate the note for Blast markers and put it in the first new bullet, thus putting it where it is most relevant and minimising the mental 'overhead' of notes.
4. And other stuff probably, this is long enough.

The 'critical hit roll' thing is a general point, not an argument about what specific words you did or didn't use. I'm saying that the bullets tie in with the later paragraph to a degree, and one has to be careful in the later paragraph not to be ambiguous about critical hits and critical damage. Sometimes that gets a bit wordy. Perhaps it can be rationalised more, but it's not as straightforward as you're saying, and it all takes time so sometimes "clearer than it was" is good enough for the moment.

And yes, GW were not consistent about critical hits and critical damage. The actual thing is critical damage [War engine critical damage](https://thehobby.zone/resources/e40k-compendium/Content/SpecialRules/WarEngines/WarEngineCriticalDamage.htm). Potentially, if we defined it properly and were really consistent, 'critical hit' could mean 'a hit which also causes critical damage either automatically or because you rolled a 6 for it per the critical damage rules', maybe -- but that's a tangent to the main thrust here.

So, I'd like to get onto new content at some point too - but I'll always be behind you a bit because the bulk of my word-smithing logically must come after the bulk of your rules-lawyering, plus sometimes I can only find time to dip in and out. I've checked off most of the files in this PR, but still have some to do. And when I do, you can and should check that I haven't broken any important meaning -- but, so long as the meaning is correct, try not to focus overmuch on my particular choice of words or structure or this will be slow.

Reply:
I got distracted a bit there, but what I'm trying to say in relation to the particular lines you clarified that you really meant to focus on, the Save text alone wouldn't connect well with the whole 'unsaved hits, critical damage' bit. Hence the bullets, to presage that. And in itself, this is not an uncommon device in GW rules either.
Anyway, we can change in future if we have a better way - I was just trying to explain why I change this sort of thing so we don't have to debate it every time.

Reply:
I think we're still talking at cross-purposes, but please mark it as resolved.